js/scripts.js

```diff
@@ -0,0 +1,12 @@
+
+var buttonElem = document.getElementById('addElem');
+var list = document.getElementById('list');
+
+
+buttonElem.addEventListener('click', function() {
+	var newListItem = document.createElement('li');
+	var getListItem = document.getElementsByTagName('li');
+	var numberListItem = getListItem.length;
```

Comment:
Oki - działa.
Chociaż to ;)

BTW - w instrukcji kodilli jest tak:
list.innerHTML += '<li>item</li>'

Jak dam '+=" i jak dam samo "=", to nie widzę różnicy - pewnie jest jakaś?